Controller/CRUDController.php

```diff
@@ -105,7 +105,7 @@ public function listAction()
         $formView = $datagrid->getForm()->createView();
 
         // set the theme for the current Admin Form
-        $this->get('twig')->getExtension('form')->renderer->setTheme($formView, $this->admin->getFilterTheme());
+        $this->get('twig')->getExtension('Symfony\Bridge\Twig\Extension\FormExtension')->renderer->setTheme($formView, $this->admin->getFilterTheme());
```

Comment:
Ain't it the same logic like symfony form names? They changed from key to class name.

Not sure if old symfony version still need the need the `key` instead of a `class name`. Have you checked this?